tests/tfhe_rust_bool/end_to_end_fpga/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "heir-tfhe-rust-integration-test"
+version = "0.1.0"
+edition = "2021"
+default-run = "main"
+
+[dependencies]
+clap = { version = "4.1.8", features = ["derive"] }
+rayon = "1.6.1"
+serde = { version = "1.0.152", features = ["derive"] }
+tfhe = { path = "tfhe-rs/tfhe", features = [
```

Comment:
nit: Could you add a quick comment in the README.md that says that the tfhe library is expected to be a local path dependency? I assume even if I'm using version "X" of the upstream tfhe without the fpga feature, that I should still be able to run locally on my CPU so long as there are no packed operations, right? Otherwise, just specify that this expects a tfhe-rs fork that has a batched API.